PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -91,6 +91,17 @@ public SparkSubmitModel(@NotNull Project project, @NotNull SparkSubmissionParame
         setTableModel(new SubmissionTableModel(submissionParameter.flatJobConfig()));
```

Comment:
Just call `this(project, new SparkSubmissionParameter(), new SparkSubmitAdvancedConfigModel(), new SparkSubmitJobUploadStorageModel());`please. 

Or we can add a builder sub class for this class.